api/src/test/java/org/openmrs/module/eptsreports/reporting/intergrated/utils/ResumoMensalLiveTest.java

```diff
@@ -113,14 +113,21 @@ public void B6() throws EvaluationException {
 
   @Test
   public void B8() throws EvaluationException {
-    CohortDefinition cd = resumoMensalCohortQueries.getPatientsWhoDied();
+    CohortDefinition cd = resumoMensalCohortQueries.getPatientsWhoDied(true);
```

Comment:
Live tests are normally not necessary, but integration tests are essential.

Reply:
Understood, working on the integration tests now.